bin/build.js

```diff
@@ -38,9 +42,10 @@ if (isDevEnv) {  // don't minify in dev
 } else {
     console.log('dev src length:', src.length);
 
-    const result = ugly.minify(src);
+    const result = ugly.minify(uglySrc);
     final_code = result.code;
     if (result.error) {
+        console.error(result);
```

Comment:
is there a reason you are printing the entire result and not just the `result.error`?

Reply:
Last I checked when there was an error result object didn't have any attributes but `error` since we are making the process more verbose I assumed having the full picture if it doesn't cover the useful portions, can be helpful. printing just `result.error` is fine too and easy to change

Reply:
if there is no other meaningful info in the `result` object, I would just print the `error` field. No need for extra text with provides no useful meaning